tracim/tests/functional/test_session.py

```diff
@@ -1,57 +1,73 @@
 # coding=utf-8
-import pytest
-
 from tracim.tests import FunctionalTest
+from tracim.tests import FunctionalTestNoDB
 
 
 class TestLogoutEndpoint(FunctionalTest):
 
-    def test_logout(self):
+    def test_api__access_logout_get_enpoint__ok__nominal_case(self):
+        res = self.testapp.post_json('/api/v2/sessions/logout', status=204)
+
+    def test_api__access_logout_post_enpoint__ok__nominal_case(self):
         res = self.testapp.get('/api/v2/sessions/logout', status=204)
 
 
+class TestLoginEndpointUnititedDB(FunctionalTestNoDB):
+
+    def test_api__try_login_enpoint__err_500__no_inited_db(self):
+        params = {
+            'email': 'admin@admin.admin',
+            'password': 'admin@admin.admin',
+        }
+        res = self.testapp.post_json(
+            '/api/v2/sessions/login',
+            params=params,
+            status=500,
+        )
+
```

Comment:
add assert of error content (in  `res.body` ou `res.json`)
Example : 

```
assert is_instance(dict, res.json)
assert 'code' in res.json.keys()
assert 'msg' in res.json.keys()
assert 'detail' in res.json.keys()


```

Reply:
#55.